circle.yml

```diff
@@ -0,0 +1,10 @@
+## Customize dependencies
+dependencies:
+  override:
+    - npm prune
```

Comment:
@mastilver Do you mean replace both commands with update, or replace install with update? I added prune because I once ran into an edge case (a rare one, admittedly) where a removed dependency was not uninstalled, and broke the install command due to unmet peer dependencies.